book/website/project-design/sdp/types.md

```diff
@@ -11,3 +11,9 @@ These are versions of inner source working in which open collaboration and open
 Therefore each research project needs to find the best solution for their needs.
 
 The following sub-chapters give an overview of the types of sensitive data and metadata that need to be identified in datasets and therefore handled securely in research projects.
+
+Or you could watch this video below that has been created by ELIXIR-UK as an introduction to sensitive data.
```

Comment:
```suggestion
Here is a short video introducing sensitive data by ELIXIR-UK.
```

Reply:
Looks good to me, https://deploy-preview-2850--the-turing-way.netlify.app/project-design/sdp/types.html.